configs/default/HARC-HAKRCF405ICM.config

```diff
@@ -0,0 +1,129 @@
+# Betaflight / STM32F405 (S405) 4.2.4
+
+#define USE_GYRO_SPI_ICM42688P 
+#define USE_ACC_SPI_ICM42688P
+#define USE_FLASH_W25Q128FV
+#define USE_MAX7456
+
+board_name HAKRCF40ICM
+manufacturer_id HARC
+
+# resources
+resource BEEPER 1 C13
+resource MOTOR 1 C08
+resource MOTOR 2 C09
+resource MOTOR 3 A08
+resource MOTOR 4 A09
+resource MOTOR 5 A10
+resource MOTOR 6 B04
+resource MOTOR 7 B00
+resource MOTOR 8 B01
+resource LED_STRIP 1 B03
+resource SERIAL_TX 1 B06
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 B10
+resource SERIAL_TX 4 A00
+resource SERIAL_TX 5 C12
+resource SERIAL_TX 6 C06
+resource SERIAL_RX 1 B07
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 3 B11
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 5 D02
+resource SERIAL_RX 6 C07
+resource INVERTER 1 C00
+resource I2C_SCL 1 B08
+resource I2C_SDA 1 B09
+resource LED 1 C15
+resource LED 2 C14
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 C10
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 C11
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 B05
+resource ADC_BATT 1 C01
+resource ADC_CURR 1 C03
+resource PINIO 1 C02
+resource PINIO 2 C05
+resource FLASH_CS 1 A15
+resource OSD_CS 1 B12
+resource GYRO_EXTI 1 C04
+resource GYRO_CS 1 A04
+
+
+# timer
+timer C08 AF3
+# pin C08: TIM8 CH3 (AF3)
+timer C09 AF3
+# pin C09: TIM8 CH4 (AF3)
+timer A08 AF1
+# pin A08: TIM1 CH1 (AF1)
+timer A09 AF1
+# pin A09: TIM1 CH2 (AF1)
+timer A10 AF1
+# pin A10: TIM1_CH3 (AF1)
+timer B04 AF2
+# pin B04: TIM3_CH1 (AF2)
+timer B00 AF2
+# pin B00: TIM3 CH3 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer B03 AF1
+# pin B03: TIM2 CH2 (AF1)
+timer B08 AF2
+# pin B08: TIM4 CH3 (AF2)
+timer B09 AF3
+# pin B09: TIM11 CH1 (AF3)
+
+# dma=
+dma ADC 3 0
+# ADC 3: DMA2 Stream 0 Channel 2
+dma pin C08 0
+# pin C08: DMA2 Stream 2 Channel 0
+dma pin C09 0
+# pin C09: DMA2 Stream 7 Channel 7
+dma pin A08 0
+# pin A08: DMA2 Stream 6 Channel 0
+dma pin A09 0
+# pin A09: DMA2 Stream 6 Channel 0
+dma pin A10 0
+# pin A10: DMA2 Stream 6 Channel 0
+dma pin B04 0
+# pin B04: DMA1 Stream 4 Channel 5
+dma pin B00 0
+# pin B00: DMA1 Stream 7 Channel 5
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin B08 0
+# pin B08: DMA1 Stream 7 Channel 2
+dma pin B03 0
+# pin B03: DMA1 Stream 6 Channel 3
+
+# feature
+feature RX_SERIAL 
+feature OSD
```

Comment:
OSD is user defined in cloud builds.

Reply:
This turns it on as a feature not as a define ? With out thia it won’t be be on when you flash the fc.. sorry was copying other targets

Reply:
#define USE_OSD does not enable feature OSD so this needs to stay.

Reply:
Ok, thought this was defined centrally instead of in each board definition